packages/rx_bloc_cli/mason_templates/bricks/lib_dev_menu/__brick__/.run/development_with_dev_menu.run.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Do we actually need to add new run configurations?
We have quite a lot of them already. Adding even more might only pollute the project with something that will not be used that often.

As a rule of thumb I think it is a good idea to add new run configuration for things that we know for sure will be used on a daily basis and not stuff that you will need 2 times in the month.

I think for this case we can just document the dart define in readme.md so the developers know how to use it. Wew can even add two more lines of text explaining how to add it to the existing run configurations.

@StanevPrime @PrimeSwiftArt what is your opinion on this?

Reply:
To be honest I do not understand why we even need two flags for enabling the dev menu. See the comments below.